packages/angular-cli/commands/init.run.ts

```diff
@@ -1,13 +1,29 @@
 import * as chalk from 'chalk';
 import LinkCli from '../tasks/link-cli';
 import NpmInstall from '../tasks/npm-install';
+import {oneLine, stripIndent} from 'common-tags';
 
 const Promise = require('../ember-cli/lib/ext/promise');
 const SilentError = require('silent-error');
-const validProjectName = require('../ember-cli/lib/utilities/valid-project-name');
 const normalizeBlueprint = require('../ember-cli/lib/utilities/normalize-blueprint-option');
 const GitInit = require('../tasks/git-init');
 
+const packageNameRegexp = /^[a-zA-Z][.0-9a-zA-Z]*(-[a-zA-Z][.0-9a-zA-Z]*)*$/;
```

Comment:
If this is essentially duplicated code, it's better to move it to a separate module similar to the deleted ember one.